src/CharNames.java

```diff
@@ -8,87 +9,84 @@
         }
+    }
 
-        /**
-         * method to iterate through map and print the key-values
-         */
-        public void printMapValue(String aClass)
-        {
-            if(charCatalog.containsKey(aClass)){
-                System.out.println(charCatalog.get(aClass));
-            } else {
-                System.out.println("The character class " + aClass + " does not exist");
-            }
+    /**
+     * method to iterate through map and print the key-values
+     */
+    public void printMapValue(String aClass) {
+        if (charCatalog.containsKey(aClass)) {
+            System.out.println(charCatalog.get(aClass));
+        } else {
+            System.out.println("The character class " + aClass + " does not exist");
         }
+    }
 
-        /**
-         * method for adding an entry to the map
-         * Requires a key(character class) and a set of names(characters
-         * that fit the class criteria)
-         */
-        public void addMapEntry(String aClass, Set charNames)
-        {
-            if(charCatalog.containsKey(aClass)){
-                charCatalog.put(aClass, charNames);
-            } else {
-                charCatalog.put(aClass, charNames);
-            }
+    /**
+     * method for adding an entry to the map
+     * Requires a key(character class) and a set of names(characters
+     * that fit the class criteria)
+     */
+    public void addMapEntry(String aClass, Set charNames) {
+        if (charCatalog.containsKey(aClass)) {
+            charCatalog.put(aClass, charNames);
+        } else {
+            charCatalog.put(aClass, charNames);
         }
+    }
 
-        /**
-         * deletes a name from a specific character class
-         */
-        public void deleteFromValueAtKey(String className, String charName)
+    /**
+     * deletes a name from a specific character class
+     */
+    public void deleteFromValueAtKey(String className, String charName) {
+        if (charCatalog.get(className).contains(charName))
         {
-            if(charCatalog.get(className).contains(charName)){
-                charCatalog.get(className).remove(charName);
-            }
+            charCatalog.get(className).remove(charName);
         }
+    }
+
+    public void addToValueAtKey(String className, String charName)
+    {
+        charCatalog.get(className).add(charName);
+    }
 
-        public void addToValueAtKey(String className, String charName)
-        {
-            charCatalog.get(className).add(charName);
-        }
     /**
      * Deletes key-pair value of a class by taking the clas type,
      * returning true if it get deleted, flase otherwise.
      */
-    public boolean deleteEntry(String theClass) {
-        if (charCatalog.containsKey(theClass)) {
+    public boolean deleteEntry(String theClass)
+    {
+        if (charCatalog.containsKey(theClass))
+        {
             charCatalog.remove(theClass);
```

Comment:
if you look at the docs for 'remove' you'll see it says

> The method returns the value that was previously mapped to the specified key if the key exists else the method returns NULL.

so you could re-write this a bit to be simpler if you wanted ;)